### PR TITLE
docs(weekly): add this week in databend 86

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+Do you remember how to register scalar functions in Databend? You can check [Doc | How to Write a Scalar Function](https://databend.rs/doc/contributing/how-to-write-scalar-functions) and [PR | #10737](https://github.com/datafuselabs/databend/pull/10737) to verify your answer.
+
+## Highlights
+
+Here are some noteworthy items recorded here, perhaps you can find something that interests you.
+
+- Learn how to monitor Databend using Prometheus and Grafana by reading our new document. *[Doc | Monitor - Prometheus & Grafana](https://databend.rs/doc/monitor/prometheus-and-granafa)*
+- If you want to connect Metabase with Databend and create dashboards, we have developed the [Metabase Databend Driver](https://github.com/databendcloud/metabase-databend-driver/releases/latest) and provided simple steps. *[Doc | Integrations - Metabase](https://databend.rs/doc/integrations/gui-tool/metabase)*
+- Databend now supports `PIVOT`, `UNPIVOT`, `GROUP BY CUBE` and `GROUP BY ROLLUP` query syntax. For more information, please see PR [#10676](https://github.com/datafuselabs/databend/pull/10676) and [#10601](https://github.com/datafuselabs/databend/pull/10601).
```
Comment (PsiACE) [inline website/blog/2023-03-24-databend-weekly-86.md:122]: cc @soyeric128 These query syntaxes do not have documentation yet.

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### FlightSQL Handler in Progress
+
+Flight SQL is an innovative open database protocol that caters to modern architectures. It boasts a columnar-oriented design and provides seamless support for parallel processing of data partitions.
+
+The benefits of supporting FlightSQL include reducing serialization and deserialization during query execution, as well as easily supporting SDKs in different languages using predefined proto.
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:48]: using predefined proto.-> using a predefined protocol.

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+Databend is actively working on supporting the FlightSQL Handler. If you're interested, visit the following materials.
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:50]: We're currently engaged in developing support for the FlightSQL Handler. If you're interested, refer to the following links:

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+- [Issue #10745 | tracking: FlightSQL handler](https://github.com/datafuselabs/databend/issues/10745)
+- [PR #10732 | feat: basic impl for FlightSQL handler](https://github.com/datafuselabs/databend/pull/10732).
+
+### Natural Language to SQL
+
+Are you still struggling with how to write complex queries? By integrating with popular AI services, Databend provides you with an efficient built-in solution - the `AI_TO_SQL` function.
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:57]: By integrating with the popular AI services, Databend now provide you an efficient built-in solution - the `AI_TO_SQL` function.

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+*************************** 1. row ***************************
+     database: openai
+generated_sql: SELECT name, SUM(price) AS total_spent, COUNT(order_id) AS total_orders
+               FROM users
+                        JOIN orders ON users.id = orders.user_id
+               WHERE country = 'USA' AND age > 30 AND order_date BETWEEN '2022-01-01' AND '2022-12-31'
+               GROUP BY name;
+```
+
+If you want to quickly experience our natural language to SQL function, you can try [deploying Databend](https://databend.rs/doc/deploy) or [registering for our cloud service](https://app.databend.com/). In addition, reading related materials can also help you learn more information:
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:74]: The function is now available on both Databend and Databend Cloud. To learn more about how it works, refer to the  following links:

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+- [Doc | AI Functions - AI_TO_SQL](https://databend.rs/doc/sql-functions/ai-functions/ai-to-sql)
+- [Blog | Databend Understands You Better with OpenAI](https://databend.rs/blog/ai2sql)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Vector Similarity Calculation in Databend
+
+Databend adds a new function called `cosine_distance`. This function accepts two input vectors, `from` and `to`, which are represented as slices of f32 values.
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:85]: Databend adds a new function called `cosine_distance`. 

Already ?   If so, adds -> has added

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+}
+```
+
+Do you remember how to register scalar functions in Databend? You can check [Doc | How to Write a Scalar Function](https://databend.rs/doc/contributing/how-to-write-scalar-functions) and [PR | #10737](https://github.com/datafuselabs/databend/pull/10737) to verify your answer.
+
+## Highlights
+
+Here are some noteworthy items recorded here, perhaps you can find something that interests you.
+
+- Learn how to monitor Databend using Prometheus and Grafana by reading our new document. *[Doc | Monitor - Prometheus & Grafana](https://databend.rs/doc/monitor/prometheus-and-granafa)*
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:120]: - Learn how to monitor Databend using Prometheus and Grafana: *[Doc | Monitor - Prometheus & Grafana](https://databend.rs/doc/monitor/prometheus-and-granafa)*

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+- If you want to connect Metabase with Databend and create dashboards, we have developed the [Metabase Databend Driver](https://github.com/databendcloud/metabase-databend-driver/releases/latest) and provided simple steps. *[Doc | Integrations - Metabase](https://databend.rs/doc/integrations/gui-tool/metabase)*
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:121]: [Metabase Databend Driver](https://github.com/databendcloud/metabase-databend-driver/releases/latest) helps you connect Databend to Metabase and dashboard your data: [Doc | Integrations - Metabase](https://databend.rs/doc/integrations/gui-tool/metabase)*

File: website/blog/2023-03-24-databend-weekly-86.md (hunk 1 above)
Comment (soyeric128) [reply website/blog/2023-03-24-databend-weekly-86.md:122]: Thank you 4 letting me know.

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+- Databend now supports `PIVOT`, `UNPIVOT`, `GROUP BY CUBE` and `GROUP BY ROLLUP` query syntax. For more information, please see PR [#10676](https://github.com/datafuselabs/databend/pull/10676) and [#10601](https://github.com/datafuselabs/databend/pull/10601).
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Enable `-Zgitoxide` to Speed up Git Dependencies Download
+
+To accelerate the download of our git dependencies, we can activate `-Zgitoxide`. This option is significantly faster than using Git alone.
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:130]: Enabling `-Zgitoxide` can speed up the download of our Git dependencies significantly, which is much faster than using Git only.

File: website/blog/2023-03-24-databend-weekly-86.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+
+This feature integrates cargo with [gitoxide](https://github.com/Byron/gitoxide), which is a pure Rust implementation of Git that is idiomatic, lean, fast and safe.
```
Comment (soyeric128) [inline website/blog/2023-03-24-databend-weekly-86.md:132]: This feature integrates cargo with [gitoxide](https://github.com/Byron/gitoxide), a pure Rust implementation of Git that is idiomatic, lean, fast, and safe.

**Pull request merged**